### PR TITLE
feat: adds util plan to setup aws agentless workload scanning

### PR DESCRIPTION
**Pull request by gabeobrien**

### CODE REVIEW

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  version = "~> 0.6"
+
+  providers = {
+    aws = aws.usw2
+  }
+
+  regional                = true
+  global_module_reference = module.lacework_aws_agentless_scanning_global
+  // In this example the default VPC CIDR block is customized for this region.
+  vpc_cidr_block          = "10.10.34.0/24"
```
Comment (ipcrm) [inline util/tenant_setup/terraform/lacework/agentless/aws/main.tf:66]: Do we need this?

File: plans/util/aws_agentless_workload_setup.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+    post_tenant_setup:
```
Comment (ipcrm) [inline plans/util/aws_agentless_workload_setup.yaml:2]: I think we need to update this deployment name, or do we want to add a step to the existing deployment (in  state)

**Review by ipcrm — CHANGES_REQUESTED** [review]
A few questions for you!

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf (hunk 1 above)
Comment (gabeobrien) [reply util/tenant_setup/terraform/lacework/agentless/aws/main.tf:66]: The cidr block is listed on the docs:
https://docs.lacework.com/onboarding/aws-integrate-agentless-workload-scanning-with-terraform
Happy to remove it if think it is not needed?

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+provider "aws" {
+  alias  = "usw2"
+  region = "us-west-2"
+}
```
[the remark this answers is not in the document]
Comment (gabeobrien) [reply util/tenant_setup/terraform/lacework/agentless/aws/main.tf]: Done

File: plans/util/aws_agentless_workload_setup.yaml (hunk 2 above)
Comment (gabeobrien) [reply plans/util/aws_agentless_workload_setup.yaml]: Done

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf (hunk 1 above)
Comment (ipcrm) [reply util/tenant_setup/terraform/lacework/agentless/aws/main.tf:77]: It has a comment that says:

```// In this example the default VPC CIDR block is customized for this region.```

So I don't think it's needed. TF docs also show the same. Just thinking the bare minimum we can supply the better; but if there's any concern we can leave

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+provider "aws" {
+  alias  = "usw1"
+  region = "us-west-1"
+}
+
+provider "aws" {
+  alias  = "usw2"
+  region = "us-west-2"
+}
```
Comment (ipcrm) [inline util/tenant_setup/terraform/lacework/agentless/aws/main.tf:40]: Now that you've added these, you'll need to add new region blocks for each. Right now we've only got use1 and usw2 covered. Also for `use1` -> do we want an alias on that? That should be the global provider config i think?

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf (hunk 1 above)
Comment (gabeobrien) [reply util/tenant_setup/terraform/lacework/agentless/aws/main.tf]: I removed the line as now that we have 4 regions I didn't want to have conflicts.  Deploying seemed to work just fine without it.

File: util/tenant_setup/terraform/lacework/agentless/aws/main.tf (hunk 4 above)
Comment (gabeobrien) [reply util/tenant_setup/terraform/lacework/agentless/aws/main.tf]: Done

**Review by ipcrm — APPROVED** [review]
:shipit:

**Pull request merged**